### DISCUSSION TITLE
21.6.1

### DISCUSSION THREAD

**Discussion by davidjgraph** [jgraph/drawio-desktop#1371] [General]
## Releases Notes for 21.6.1
[Windows Installer](https://github.com/jgraph/drawio-desktop/releases/download/v21.6.1/draw.io-21.6.1-windows-installer.exe)
[Windows No Installer](https://github.com/jgraph/drawio-desktop/releases/download/v21.6.1/draw.io-21.6.1-windows-no-installer.exe)
[macOS - Universal](https://github.com/jgraph/drawio-desktop/releases/download/v21.6.1/draw.io-universal-21.6.1.dmg)
Linux - [deb](https://github.com/jgraph/drawio-desktop/releases/download/v21.6.1/drawio-amd64-21.6.1.deb), [snap](https://snapcraft.io/drawio), [AppImage](https://github.com/jgraph/drawio-desktop/releases/download/v21.6.1/drawio-x86_64-21.6.1.AppImage) or [rpm](https://github.com/jgraph/drawio-desktop/releases/download/v21.6.1/drawio-x86_64-21.6.1.rpm)
[Google Chrome OS](https://chrome.google.com/webstore/detail/drawio-desktop/pebppomjfocnoigkeepgbmcifnnlndla/reviews)

Windows intel x32 releases are marked -ia32-

**ChangeLog:**

- Updates to [draw.io core 21.6.1](https://github.com/jgraph/drawio/blob/v21.6.1/ChangeLog). All changes from 21.5.2 to 21.6.1 are added in this build.

<hr /><em>This discussion was created from the release <a href='https://github.com/jgraph/drawio-desktop/releases/tag/v21.6.1'>21.6.1</a>.</em>

**Comment by YajingXu** [comment]
Why can't I save a file in Windows client and it says "Invalid file data".
Thank you in advance.
